operator/pkg/operator.go

```diff
@@ -137,17 +137,18 @@ func (o *Operator) ProcessNewTaskCreatedLog(newTaskCreatedLog *servicemanager.Co
 	var proof []byte
 	var err error
 
+	// TODO: Add support for multiple chunks
```

Comment:
is this referred to the calldata case? should we raise an issue?

Reply:
Old comment